.eslintrc.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  extends: ['rollup', 'plugin:import/typescript'],
+  extends: ['rollup', 'plugin:import/typescript', 'prettier'],
```

Comment:
given, https://github.com/rollup/plugins/pull/787/files#r568671377, let's remove the `prettier` preset. I'd rather have us set our rules up rather than battling defaults from another package.